tokenserver-db-postgres/src/models.rs

```diff
@@ -618,13 +618,6 @@ impl Db for TokenserverPgDb {
         TokenserverPgDb::get_users(self, params).await
     }
 
-    async fn get_or_create_user(
-        &mut self,
-        params: params::GetOrCreateUser,
-    ) -> Result<results::GetOrCreateUser, DbError> {
-        TokenserverPgDb::get_or_create_user(self, params).await
-    }
```

Comment:
Infinite call loop from trait impl delegation.

Reply:
Good call. This was missed when we moved the `get_or_create_user` call into `tokenserver-db-common` since it was identical for MySQL and Postgres.